src/Modal/Modal.js

```diff
@@ -133,6 +135,7 @@ Modal.propDescriptions = {
     localizedTextShape: {
         closeButton: 'Aria-label for <button> element.'
     },
+    onClose: 'Callback function passing event when close button is clicked.',
```

Comment:
I know this already existed and we're just documenting it now, but should we have them all match and make it `onCloseClicked`

Reply:
This has actually made this a breaking change.  I know we're still on version 0, so that's good, but there are now docs pages that don't work correctly (_e.g._ Modal, Popover) because although `onClose` was never a documented callback, it _was_ being supported internally with `Modal`.